src/components/appTile/AppTile.js

```diff
@@ -0,0 +1,221 @@
+
+const styles = (theme) => ({
+    card: {
+        minWidth: 300,
+        maxWidth: 400,
+        minHeight: 120,
+        maxHeight: 120,
+        margin: 15,
+        padding: 5,
+        cursor: "pointer",
+        "&:hover": {
+            border: "1px solid",
+        },
+    },
+    selectedCard: {
+        border: "1px solid",
+        backgroundColor: palette.lightGray,
+    },
+    avatar: {
+        float: "left",
+        margin: 10,
+    },
+    type: {
+        textAlign: "center",
+        fontSize: 10,
+    },
+    name: {
+        margin: 10,
+        fontSize: 12,
+        textOverflow: "ellipsis",
+        overflow: "hidden",
+        whiteSpace: "nowrap",
+        display: "inline-block",
+        maxWidth: 150,
+        "&:hover": {
+            textDecoration: "underline",
+            cursor: "pointer",
+        },
+    },
+    more: {
+        float: "right",
+        height: 120,
+    },
+    creator: {
+        margin: 10,
+        top: 50,
+        fontSize: 12,
+        fontStyle: "italic",
+        textOverflow: "ellipsis",
+        overflow: "hidden",
+        whiteSpace: "nowrap",
+        display: "inline-block",
+        maxWidth: 150,
+    },
+    status: {
+        position: "relative",
+        top: 40,
+        right: -10,
+    },
+    statusNoMenu: {
+        position: "relative",
+        top: 90,
+        right: 10,
+    },
+    rating: {
+        width: "80%",
+    },
+});
+
+function AppTile(props) {
+    const {
+        classes,
+        uuid,
+        name,
+        creator,
+        rating,
+        type,
+        isPublic,
+        isBeta,
+        isDisabled,
+        isExternal,
+        onRatingChange,
+        onDeleteRatingClick,
+        MenuItems,
+        intl,
+        onAppNameClicked,
+        onAppSelected,
+        selected,
+    } = props;
+
+    const {
+        average: averageRating,
+        user: userRating,
+        total: totalRating,
+    } = rating;
+
+    const [anchorEl, setAnchorEl] = useState("");
+    const open = Boolean(anchorEl);
+
+    let avatarImgSrc = GRAVATAR_URL + md5(uuid) + "?" + GRAVATAR_OPTIONS;
+
+    return (
+        <Paper
+            className={
+                selected
+                    ? classnames(classes.card, classes.selectedCard)
+                    : classes.card
+            }
+            onClick={onAppSelected}
+        >
+            <div className={classes.avatar}>
+                <div>
+                    <img src={avatarImgSrc} alt="avatar image" />
+                </div>
+                <div className={classes.type}>{type.toLowerCase()}</div>
+            </div>
+            <div>
+                <div
+                    title={name}
+                    className={classes.name}
+                    onClick={onAppNameClicked}
+                >
+                    {name}
+                </div>
+                <div className={classes.more}>
+                    {MenuItems != null && (
```

Comment:
```suggestion
                    {MenuItems && (
```
This might be better, especially if `ESLint` will complain that we're not using `!==` here.